src/Reflection/ReflectionProperty.php

```diff
@@ -67,7 +67,7 @@ private function __construct()
     {
     }
 
-    public static function export() : void
+    public static function export()
```

Comment:
This is indeed `void`

Reply:
This method inherits from a method that has a non-`void` return type